pyserini/prebuilt_index_info.py

```diff
@@ -2224,6 +2224,35 @@
         "unique_terms": 6517412,
         "downloaded": False
     },
+    "miracl-v1.0-de": {
+        "description": "Lucene index for MIRACL v1.0 (German).",
+        "filename": "lucene-index.miracl-v1.0-de.20221004.2b2856.tar.gz",
+        "readme": "lucene-index.miracl-v1.0.20221004.2b2856.README.md",
+        "urls": [
+            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.miracl-v1.0-de.20221004.2b2856.tar.gz"
+        ],
+        "md5": "a40d1b9429c450b2e476d1e4ba22784d",
+        "size compressed (bytes)": 8708219012,
+        "total_terms": 581583743,
+        "documents": 15866222,
+        "unique_terms": 6288858,
+        "downloaded": False
+    },
+    "miracl-v1.0-yo": {
+        "description": "Lucene index for MIRACL v1.0 (Yoruba).",
+        "filename": "lucene-index.miracl-v1.0-yo.20221004.2b2856.tar.gz",
+        "readme": "lucene-index.miracl-v1.0.20221004.2b2856.README.md",
+        "urls": [
+            "https://vault.cs.uwaterloo.ca/s/6S68PqPxyz4GoJt/download",
+            # "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.miracl-v1.0-yo.20221004.2b2856.tar.gz"
```

Comment:
just making sure this wasn't commented out by mistake

Reply:
nope it's a url we'll need to fix after Jimmy's back, using my private URL temporarily now